index.bs

```diff
@@ -1396,18 +1528,22 @@ XRInputSourceEvent {#xrinputsourceevent-interface}
 interface XRInputSourceEvent : Event {
   readonly attribute XRFrame frame;
   readonly attribute XRInputSource inputSource;
+  readonly attribute long buttonIndex;
```

Comment:
Should this be nullable?

Reply:
That's probably a better pattern than -1, yeah. Thanks!